app/feature/feature-payments/src/main/kotlin/com/hedvig/android/feature/payments/navigation/PaymentsGraph.kt

```diff
@@ -91,13 +96,33 @@ fun NavGraphBuilder.paymentsGraph(
       )
     }
 
+    composable<PaymentsDestinations.Forever>(
+      deepLinks = listOf(
+        navDeepLink { uriPattern = hedvigDeepLinkContainer.forever },
+      ),
```

Comment:
```suggestion
```
I think we should not have the deep link on this one for forever. Instead the deep link should still just take the member to the top-level destination which shows the Forever information

Having the same deep link at two spots will most likely result in undefined behavior, like the one winning being the one first specified in the graph or something silly like that, which we definitely do not want

Reply:
shoud've thought of that:)

Reply:
Dw about it, that's what the PR review is for 😊